type-alias-axon-serializer/src/main/java/org/alias/axon/serializer/experimental/AliasableResourceBundleClassloader.java

```diff
@@ -48,7 +48,7 @@ public AliasableResourceBundleClassloader(ResourceBundle resourceBundle, ClassLo
 	@SuppressWarnings({ "rawtypes", "unchecked" })
     public Class findClass(String name) throws ClassNotFoundException {
 		if (!resourceBundle.containsKey(name)) {
-			throw new ClassNotFoundException();
+			return Class.forName(name);
```

Comment:
What happens, if you set this line(s) back to "return null;" and change line 89 (last line in method loadClass) to "getParent().loadClass(name);"? Would this solve the NullPointerException?
This would also enable to exchange the fallback from outside, since the parent ClassLoader can be set via the constructor.